manifests/vhost/error_documents.pp

```diff
@@ -0,0 +1,21 @@
+
+define apache::vhost::error_documents (
+  $error_documents,
+  $vhost                                                                            = $name,
```

Comment:
I know where it's coming from, but you don't need to preserve the whitespace before the `=` here.